bundles/org.openhab.binding.insteon/src/main/resources/OH-INF/config/legacy-config.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config-description:config-descriptions
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:config-description="https://openhab.org/schemas/config-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/config-description/v1.0.0 https://openhab.org/schemas/config-description-1.0.0.xsd">
+
+	<config-description uri="channel-type:insteon:legacy-button">
```

Comment:
See other comment, not sure if this is going to work. Existing configurations for managed items AFAIK not updated within the json db.

Reply:
As mentioned above, since the channels are recreated including the existing configuration during the migration, I don't think this will apply unless I am missing something.

Reply:
Isn't `config-description` used to how channel configurations are displayed in MainUI and not how it is stored?

I couldn't find any reference to these in my json db. Maybe I am missing something.